### PR TITLE
[FIX] account: fix dashboard graphs alignment

### PR DESCRIPTION
**Pull request by clbr-odoo**
Graphs were not aligned depending on data. They will now stick to bottom of the card and fill consistently the space above.

Task: 3071124

### CODE REVIEW

File: addons/account/views/account_journal_dashboard_view.xml [hunk 1]
```diff
@@ -29,7 +29,9 @@
                                         <t t-call="HasSequenceHoles"/>
                                     </div>
                                 </div>
-                                <t t-if="journal_type == 'bank' || journal_type == 'cash' || journal_type == 'sale' || journal_type == 'purchase'" t-call="JournalBodyGraph"/>
+                                <div class="row mt-auto">
+                                    <t t-if="journal_type == 'bank' || journal_type == 'cash' || journal_type == 'sale' || journal_type == 'purchase'" t-call="JournalBodyGraph"/>
```
Comment (FlorianGilbert) [inline addons/account/views/account_journal_dashboard_view.xml:33]: I know that's not the purpose of this task but `==` could be change for `===` or you can do something like that:
```suggestion
                                    <t t-if="['bank', 'cash', 'sale', 'purchase'].includes(journal_type)" t-call="JournalBodyGraph"/>
```

File: addons/account/views/account_journal_dashboard_view.xml [hunk 2]
```diff
@@ -29,7 +29,9 @@
...
+                                </div>
                             </div><div class="container o_kanban_card_manage_pane dropdown-menu" role="menu">
```
Comment (FlorianGilbert) [inline addons/account/views/account_journal_dashboard_view.xml:35]:
```suggestion
                            </div>
                            <div class="container o_kanban_card_manage_pane dropdown-menu" role="menu">
```

**Comment by chklop** [comment]
LGTM !

**Review by FlorianGilbert — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**